sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import co.nimblehq.sample.compose.navigation.NavigationRoute
+import co.nimblehq.sample.compose.ui.userReadableMessage
+
+@Composable
+fun HomeScreen(navController: NavController) {
+    val viewModel = hiltViewModel<HomeViewModel>()
```

Comment:
Shouldn't we pass in `viewModel` as an argument? This is quite coupled with Hilt 👀

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279